workshops/pacman game/README.md

```diff
@@ -0,0 +1,337 @@
+* **Going a step further: **If you want to take things a step further and introduce an animation, sound effect, or other prettiness factor, we would introduce it after `brick(x, y)`. Try adding `print(“PAC-MAN”)` as an example.
+
+## Giving Direction to Pac-man
+
+We created a function that lets our Pac-man move, but it doesn't know where to go. It only followed the direction that we provided to it in the `aim` variable. Let's create a function that lets our Pac-man change that `aim` value, or in other terms, change it's direction. Add this code into the bottom of your code:
+
+```python
+def change(x, y):
+    if valid(pacman + vector(x, y)):
+        aim.x = x
+        aim.y = y
+```
+
+We first check if the direction is valid (we aren't inside of a brick or anything), then we set the `aim` we initialized at the start of the code.
+
+## The Final Stretch!
+
+As we mentioned earlier, this game is coded using a principle called modular programming. This means that everything we code is in functions, now we can bring it together. We can call these functions by writing the function name and the parameters it needs in parentheses. This code block will put all the pieces together:
+
+```python
+setup(420, 420, 370, 0)
+hideturtle()
+tracer(False)
+writer.goto(160, 160)
+writer.color('white')
+writer.write(state['score'])
+listen()
+onkey(lambda: change(5, 0), 'Right')
+onkey(lambda: change(-5, 0), 'Left')
+onkey(lambda: change(0, 5), 'Up')
+onkey(lambda: change(0, -5), 'Down')
+world()
+move()
+done()
+```
+
+**Things to note:**
+
+* These are all functions we created earlier, we are just putting the pieces together.
+* The grid, as mentioned earlier, is scaled up to 420x420 (from 20x20). We use `turtle`'s built in `setup()` function to accomplish this.
+* `hideturtle()`, `listen()`, and `tracer(False)` are all default `turtle` functions that will let us remove any unnecessary default items/features and enable us to listen for keystrokes.
+  * We are picking up the user's keystrokes by using Python's built-in `onkey()` function.
+
+## Extending the game
+
+Our version of Pac-man is pretty basic. We are utilizing 2D vector graphics and Python’s turtle graphics library to draw shapes and dots on our screen. 
+
+* This art style is very much in-line with the look and feel of the original incarnation of the game in the 1980s. 
+  * If you want to take the game a step further you could introduce graphical sprites, add more details to Pac-man or ghosts like eyes, etc.
+  * You can design end numbers of levels for Pac-man by doing changes in the `tiles` variable. Different color schemes, size, number of ghosts, etc.
+
+You can also add other types of roadblocks into the app to make it more difficult, which usually incites making the game more addictive:
```

Comment:
"incites making the game" is awkward phrasing, you could probably just do "which makes the game"